recipes/callingcardstools/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "callingcardstools" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 74a20d1eb22f7bdf3373e70e57737caefe4ea63f115e975dba6c04be814288fd
+
+build:
+  noarch: generic
```

Comment:
Should this be `noarch: python`?

Reply:
yes -- changed in [427ccaf](https://github.com/bioconda/bioconda-recipes/pull/41078/commits/427ccaf1f14fabc7dd87ff0371cba1ca8222c057)